contained-innersource.md

```diff
@@ -0,0 +1,70 @@
+## Title  
+
+Contained InnerSource
+
+## Patlet
+
+Apply InnerSource methods to facilitate collaboration in a cross divisional 
+project but don't invest in soliciting contributions from outside of that
+project.
+
+## Problem  
+
+Traditional development practices do not work as effectively for cross
+divisional projects as do InnerSource style practices. However, Management
+does not support adopting InnerSource practices because it regards efforts for
+soliciting and facilitating contributions from outside of the project as
+detrimental for the required efficiency. 
```

Comment:
If management doesn't support it, shouldn't we build tools to help convince them rather than sacrifice the benefits of having internally open repositories? Will a hobbled version of InnerSource convince management of anything?

Reply:
I think it's fair to have more than one possible solution that addresses the same problem.  @NewMexicoKid would this just be a separate pattern with the same problem statement?

Reply:
Yes. There can definitely be different solutions that address the same problem or that may apply when different contexts or forces are at play.